README.md

```diff
@@ -44,9 +44,12 @@ The quickest way to start a single member cluster for development purposes is to
 [Docker images](https://hub.docker.com/r/hazelcast/hazelcast/).
 
 ```bash
-docker run -p 5701:5701 hazelcast/hazelcast:4.1.1
+docker run -p 5701:5701 hazelcast/hazelcast
 ```
 
+This command uses the `latest` tag in order to fetch the latest hazelcast version. You can find all available tags
```

Comment:
```suggestion
This command fetches the latest Hazelcast version. You can find all available tags
```

Reply:
Is it always the case? What is latest tag in hazelcast docker? Is it master snapshot?  Is it 5.0? is it 5.1-beta? or 5.1? It is updated 2 months ago. We may lead users to wrong version.

Reply:
Maybe just saying `latest available hazelcast docker version` is better

Reply:
Not a snapshot, it is just the most recently released patch version of the latest minor version. So, it should be okay to say that it is the latest Hazelcast version IMHO

Reply:
ok changing it then. I think they did not update docker hub yet after 5.1 release.